decidim-core/app/cells/decidim/participatory_space_private_user/show.erb

```diff
@@ -0,0 +1,23 @@
+<div class="profile__user">
+  <div class="profile__user-avatar-container">
+    <div class="<%= has_profile? ? "profile__user-avatar" : "profile__user-avatar !border-0" %>">
+      <%= image_tag(has_profile? ? model.avatar_url(:big) : model.non_user_avatar_path, alt: "member-avatar") %>
+    </div>
+  </div>
+  <div>
+    <div class="<%= has_profile? ? "profile__user-name" : "profile__user-name !no-underline" %>">
+      <%= name %>
+    </div>
+    <% if nickname.present? %>
+      <span class="profile__user-nick block">
+        <%= nickname %>
+      </span>
+    <% end %>
+
+    <div class="mt-3">
+      <span class="text-gray-2 capitalize font-semibold">
+        <%= translated_attribute role %>
```

Comment:
I would really love to see a spec on how would this field would behave when the role is a sql injection

Reply:
In your specs / factories, do not provide a name for the role: role: { en: :member} , but use something like `generate_localized_word(:status_name, skip_injection:)` helper ... 
This way, we could also test the page for unexpected XSS attacks

Reply:
I think I fixed everything except this one. Could you resolve the conversations that are done? Or can I do it?

Reply:
Also, done ✅